client/components/web-preview/style.scss

```diff
@@ -22,49 +40,41 @@
 }
 
 .web-preview__content {
-	box-shadow: 0 0 0 1px transparentize( lighten( $gray-dark, 20% ), .5 ),
-		0 1px 2px lighten( $gray-dark, 30% );
+	box-shadow: 0 0 0 1px rgba( $gray, 0.5 );
+	background: $gray-light;
 	border-radius: 4px 4px 0 0;
 	position: absolute;
-		top: 47px;
+		top: 24px;
 		left: 0;
 		right: 0;
 		bottom: 0;
 	margin: 0 auto;
-	transform: scale( 0.96 );
-	transition: 0.1s transform cubic-bezier( 0.25, 0.5, 0.5, 0.9 );
-	background: lighten( $gray, 30% );
+	opacity: 0;
+	transform: translateY( 80vh );
+	transition: transform 0.2s ease-out,
+		opacity 0.1s ease-in-out,
+		max-width 0.2s ease-out;
 
 	.is-computer & {
-		width: 100%;
+		max-width: 1040px;
```

Comment:
If possible, I'd try to avoid a % just because it would stretch a lot on wide screens... for roughly the same reason you expressed too. :)
Totally ok in raising it to 1200 or so. :)